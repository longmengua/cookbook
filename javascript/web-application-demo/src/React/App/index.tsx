import React from "react";
import "./index";
import "./index_c";

function App() {
	return (
		<div className="App">
			Index start
		</div>
	);
}

export default App;
