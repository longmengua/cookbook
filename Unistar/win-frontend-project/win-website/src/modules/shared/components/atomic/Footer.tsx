import React from "react";
// import "../../styles/layout/_footer.scss";
function Footer(): React.FunctionComponentElement<{}> {
    return (
        <div className="footer">
            <div className="container">
                <div className="logo">
                    <img src={require("../../../../asset/images/logo.png")} />
                </div>
                <div className="copyright">Copyright @奖源8网 2019 All Right Reserved.</div>
                <div className="help-item">
                    <ul>
                        <li>
                            <div className="help-icon01">
                                <span className="help-tip"> QQ : 213759986</span>
                            </div>
                        </li>
                        <li>
                            <div className="help-icon02">
                                <span className="help-tip">skype : +639459734806</span>
                            </div>
                        </li>
                        <li>
                            <div className="help-icon03">
                                <span className="help-tip">telegram : +639459734820</span>
                            </div>
                        </li>
                        <li>
                            <div className="help-icon04">
                                <span className="help-tip">gmail : jy8web@gmail.com</span>
                            </div>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    );
}

export default Footer;
