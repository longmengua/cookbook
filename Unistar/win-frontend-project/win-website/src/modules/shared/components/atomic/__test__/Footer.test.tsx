import "jest-enzyme";
import Footer from "../Footer";
import {shallow} from "enzyme";
import React from "react";

describe("<Footer />", (): void => {
    it("take a snapshot", (): void => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });
});
