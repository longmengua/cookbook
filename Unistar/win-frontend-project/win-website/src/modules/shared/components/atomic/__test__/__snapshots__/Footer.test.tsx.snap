// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> take a snapshot 1`] = `
<div
  className="footerf"
>
  <div
    className="container"
  >
    <div
      className="logo"
    >
      <img
        src="test-file-stub"
      />
    </div>
    <div
      className="copyright"
    >
      Copyright @奖源8网 2019 All Right Reserved.
    </div>
    <div
      className="help-item"
    >
      <ul>
        <li>
          <div
            className="help-icon01"
          >
            <span
              className="help-tip"
            >
               QQ : 1463279830
            </span>
          </div>
        </li>
        <li>
          <div
            className="help-icon02"
          >
            <span
              className="help-tip"
            >
              skype : +639459734806
            </span>
          </div>
        </li>
        <li>
          <div
            className="help-icon03"
          >
            <span
              className="help-tip"
            >
              telegram : +639459734820
            </span>
          </div>
        </li>
        <li>
          <div
            className="help-icon04"
          >
            <span
              className="help-tip"
            >
              gmail : jy8web@gmail.com
            </span>
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
